link_linux.go

```diff
@@ -2304,12 +2304,27 @@ func parseSittunData(link Link, data []syscall.NetlinkRouteAttr) {
 func addVtiAttrs(vti *Vti, linkInfo *nl.RtAttr) {
 	data := nl.NewRtAttrChild(linkInfo, nl.IFLA_INFO_DATA, nil)
 
-	ip := vti.Local.To4()
```

Comment:
I understand it is just personal preference, but we could avoid the new `family` variable (and most of all for a future reader to have to look into the function if the variable is used anywhere else) if we feel the following is clear enough:
```
ip := vti.Local
if ip.To4() != nil { 
    ip = ip.To4()
}
```
same for `vti.Remote` below

Reply:
The reason is that `family` determines both addresses family. I.e. it doesn't make sense to test family of `Remote`, as both should belong to the same family.

Would you still prefer calling `.To4()` twice?

Reply:
Yes, either ways you look at it it has its pros and cons. Just personal preference. I am fine to leave it as is too.